HW16/3args-2.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+FIRST=$1
+SECOND=$2
+THIRD=$3
+
+RE='^(-?([0-9]{1,6}|10{6})|one|two|three)$'
+
+function checkString {
+  if ! [ $# -eq 0 ]; then
+    if [[ $1 =~ $RE ]]; then
```

Comment:
What about range -1000000 to 1000000 ?

Reply:
-?([0-9]{1,6}|10{6})
Minus is optional ( (any digit 1-6 (0-999999) or 1 and six zeros (1000000) )
This regex expression gives a range -1000000 to 1000000

Reply:
Ah, too smart for me ;)

Reply:
Ah, too smart for me ;)
Well done!